### ISSUE TITLE
Swift 4 support, add @objc to some methods

### ISSUE THREAD

**Issue by loopingGIT** [whitesmith/WSTagsField#68]
I am having a couple of issues with #selector on swift 4, this is the error: 

> Argument of '#selector' refers to instance method 'onTextFieldDidChange' that is not exposed to Objective-C

I think that just adding @objc would fix this.

```
//WSTagsField.swift
@objc open func onTextFieldDidChange(_ sender: AnyObject) {
        if let didChangeTextEvent = onDidChangeText {
            didChangeTextEvent(self, textField.text)
        }
    }

//WSTagView.swift
@objc func handleTapGestureRecognizer(_ sender: UITapGestureRecognizer) {
        if let didRequestSelectionEvent = onDidRequestSelection {
            didRequestSelectionEvent(self)
        }
    }
```

**Comment by loopingGIT** [comment]
It is odd, because I can see the project here has the @objc, but that code is not being downloaded when I do a pod install.
This is the output on the console: _Using WSTagsField (2.1.2)_

Any help would be appreciated.

**Comment by krezzoid** [comment]
@loopingGIT 
`pod 'WSTagsField', git: 'https://github.com/wemap/WSTagsField.git'`

**Comment by ricardopereira** [comment]
@loopingGIT Sorry but the latest version is still using Swift 3.0.

**Comment by ricardopereira** [comment]
Swift 4 will be released when #65 has been fixed.

**Comment by loopingGIT** [comment]
@ricardopereira thanks for the comment, I saw on the commit on the files and I thought it was already out there.
